packages/vaadin-text-field/src/vaadin-text-area.js

```diff
@@ -208,11 +181,44 @@ class TextAreaElement extends ElementMixin(TextFieldMixin(ControlStateMixin(Them
 
   /** @private */
   _updateHeight() {
-    /* https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/ */
-    this._textAreaWrapper = this._textAreaWrapper || this.shadowRoot.querySelector('.textarea-wrapper');
-    this._textAreaWrapper.dataset.replicatedValue = this.inputElement.value;
-    // getComputedStyle is expensive, maybe we can use ResizeObserver in the future
-    this._dispatchIronResizeEventIfNeeded('InputHeight', getComputedStyle(this._textAreaWrapper).height);
+    const inputField = this.root.querySelector('[part=input-field]');
```

Comment:
1. Please use `this.shadowRoot` because `this.root` is Polymer specific property
2. Please consider moving `querySelector` to `ready` callback to only run once

Reply:
Done